akka-serialization-jackson/src/main/scala/akka/serialization/jackson/JacksonSerializer.scala

```diff
@@ -261,7 +264,12 @@ import akka.util.Helpers.toRootLowerCase
   }
 
   override def fromBinary(bytes: Array[Byte], manifest: String): AnyRef = {
+    fromBinary(ByteBuffer.wrap(bytes), manifest)
```

Comment:
This implementation is probably worse from a performance perspective than not implementing ByteBufferSerializer at all. I thought we would use Jackson's ByteBufferBackedOutputStream/ByteBufferBackedInputStream?

Reply:
Waiting for https://github.com/akka/akka/pull/29012 which has compression function for bytebuffer.

Reply:
I am unable to come up with a way to compress the streams. The difficulty lies in we need to prepend metadata (magic number and data length) to output stream, but they are only known when the whole compression process is done. So I am unable to write a `toBinary` function using `ByteBufferBackedOutputStream`. It is my understanding that the current `fromBinary` function of `Array[Byte]` has no performance penalty over `fromBinary` function of `ByteBuffer`. `ByteBuffer.wrap(bytes)` is only a thin wrapper of `bytes`. In order to deserialize, we always need the whole decompressed bytes in the memory. It is my understanding that this overhead is reasonable. Can you please clarify how can I improve the performance with `ByteBufferBackedOutputStream`/`ByteBufferBackedInputStream`?